Week-2/Exercise-2.1/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise 2.1</title>
+    <meta name="description" content="Pesto Plus program">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
```

Comment:
@AakashPandya The page is responsive. But there should be some text to the right of the page. Based on the responsiveness, it has to wrap down the image. I think here its by default below the image

Can you please fix it

Reply:
Updated code, so on a smaller device the right side content should stack below the image.
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/18309003/182013304-3d8a0fe0-3fa1-44ed-9d6e-e691c86a46fd.png">